.github/workflows/pr-test.yml

```diff
@@ -7,7 +7,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-18.04, ubuntu-latest, windows-2019, windows-latest]
+        os: [ubuntu-latest, windows-2019, windows-latest]
```

Comment:
We can now simplify lines 25-31 to just do `sudo apt install` from the `if` branch, we don't need the `else` branch any more.

Reply:
I thought we might just have it around so that if anyway is looking, they know to download mkl separately. But happy to remove too.

Reply:
I think the `README.md` has that part covered. It's OK to keep it here as well.

Reply:
removed. thanks!